CHANGELOG.md

```diff
@@ -56,6 +56,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 * [CVE-2022-33987] Upgrade geckodriver to 3.0.2 ([#2166](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2166))
 * Bumps percy-agent to use non-beta version ([#2415](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2415))
 * Resolve sub-dependent d3-color version and potential security issue ([#2454](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2454))
+* Bump Unset Value from 1.0.1 to 2.0.1 and Minimatch from 3.0.4 to 3.05 (#2624)(https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2624)
```

Comment:
I think it's better to match the module name, eg. unset-value, also the minimatch version missed a dot~~~
For md, the syntax to add link is https://www.markdownguide.org/basic-syntax/#links
```suggestion
* Bumps unset-value from 1.0.1 to 2.0.1 and minimatch from 3.0.4 to 3.0.5 ([#2624](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/2624))
```
nit: some context why it is needed would be helpful